packages/next/src/server/config-schema.ts

```diff
@@ -111,31 +112,42 @@ const zTurboLoaderItem: zod.ZodType<TurbopackLoaderItem> = z.union([
   }),
 ])
 
-const zTurboRuleConfigItemOptions: zod.ZodType<TurbopackRuleConfigItemOptions> =
+const zTurbopackRuleConfigItemOptions: zod.ZodType<TurbopackRuleConfigItemOptions> =
   z.object({
-    loaders: z.array(zTurboLoaderItem),
+    loaders: z.array(zTurbopackLoaderItem),
     as: z.string().optional(),
   })
 
-const zTurboRuleConfigItem: zod.ZodType<TurbopackRuleConfigItem> = z.union([
+const zTurbopackBuiltinCondition: zod.ZodType<TurbopackRuleBuiltinCondition> =
```

Comment:
```suggestion
const zTurbopackLoaderCondition: zod.ZodType<TurbopackRuleBuiltinCondition> =
```

Reply:
Similar issue as my other comment, I want to keep `builtin` in the name, but I'm happy to add `loader` as well. Otherwise it gets confusing with `zTurbopackCondition`.